packages/vue-app/types/vue.d.ts

```diff
@@ -24,10 +24,6 @@ declare module "vue/types/options" {
 
 declare module "vue/types/vue" {
   interface Vue {
-    $nuxt: {
-      $isOnline: boolean;
-      $isOffline: boolean;
-      $loading: LoadingObject;
-    };
+    $nuxt: NuxtApp
```

Comment:
And sorry here, I forgot a semicolon in the first commit of my proposal changes, so we need to add it 😁

PS : I may propose to remove semicolon and double quotes everywhere in typedefs later, but lets keep the code style already used for now

Reply:
https://github.com/kevinmarrec/nuxt.js/commit/caa3efd1763706977c98090c71c9d4994827547d 🙈

Reply:
👻